kview/src/main/java/dev/ikm/komet/kview/events/pattern/ShowPatternFormInBumpOutEvent.java

```diff
@@ -40,6 +42,8 @@ public class ShowPatternFormInBumpOutEvent extends Evt {
     public static final EvtType<ShowPatternFormInBumpOutEvent> SHOW_EDIT_FQN = new EvtType<>(DESCRIPTION_NAME, "SHOW_EDIT_FQN_DESCRIPTION");
     public static final EvtType<ShowPatternFormInBumpOutEvent> SHOW_EDIT_OTHER_NAME = new EvtType<>(DESCRIPTION_NAME, "SHOW_EDIT_OTHER_NAME_DESCRIPTION");
     private final int totalFields;
+    private final PatternField patternField;
+    private final int fieldOrder;
```

Comment:
Can we not have these global variables?

Reply:
I have added it to send the data from pattern details fields to the fields edit pane so that user can edit the values.
This is similar to the other event classes. Are we changing this implementation design across all the events? Can you please suggest the alternative ?